<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeSelectorTreeViewWidget</class>
 <widget class="QWidget" name="NodeSelectorTreeViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>12</number>
     </property>
     <item alignment="Qt::AlignVCenter">
      <widget class="QPushButton" name="bBack">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none;
    background-color: transparent;
    padding: 0px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Resources_macx.qrc">
         <normaloff>:/images/node_selector/arrow-left-ico.png</normaloff>
         <disabledoff>:/images/node_selector/arrow-left-disabled-ico.png</disabledoff>:/images/node_selector/arrow-left-ico.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignVCenter">
      <widget class="QPushButton" name="bForward">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none;
    background-color: transparent;
    padding: 0px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Resources_macx.qrc">
         <normaloff>:/images/node_selector/arrow-right-ico.png</normaloff>
         <disabledoff>:/images/node_selector/arrow-right-disabled-ico.png</disabledoff>:/images/node_selector/arrow-right-ico.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="lFolderName">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>false</bold>
         <strikeout>false</strikeout>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font-size:17px;
}</string>
       </property>
       <property name="text">
        <string notr="true">FolderName</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="syncSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lIcon">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="avatarSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lOwnerIcon">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="MegaItemTreeView" name="tMegaFolders">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QTreeView {
    show-decoration-selected: 1;
	selection-background-color: #3393df;
	 font-size: 12px;
 	qproperty-indentation: 18; 
	outline:0;
}

QTreeView::item:disabled
{
	background-color:transparent;
}

QTreeView::item:hover {
   background-color:transparent;
}

QTreeView::branch:hover {
background-color:transparent;
}

QTreeView::branch:selected {
background-color: #3393df;
}

QTreeView::item:selected {
    border: transparent;
	 color: white;
    background-color: #3393df;
}

QTreeView::branch:selected:closed:has-children:adjoins-item {
    border-image: none;
	image:url(://images/node_selector/Icon-Small-Arrow-Left-hover.png) ;
}

QTreeView::branch:selected:open:has-children:adjoins-item {
    border-image: none;
    image: url(://images/node_selector/Icon-Small-Arrow-Down-hover.png) ;
}

QTreeView::branch:!selected:closed:has-children:adjoins-item {
    border-image: none;
    image: url(://images/node_selector/Icon-Small-Arrow-Left.png) ;
}

QTreeView::branch:!selected:open:has-children:adjoins-item {
    border-image: none;
    image: url(://images/node_selector/Icon-Small-Arrow-Down.png) ;
}

QHeaderView
{
background: #FFFFFF;
font-size: 12px;
color:#576879;
padding-top: 2px;
}

QHeaderView::section {
background: #FFFFFF;
border-left:1px solid #dadada;			
border-top:0 px;
border-bottom:0px;					 					 				 					 	
border-right:0px;
padding-left:3px;
}


QHeaderView::section::first
{
padding-left: 0px;
border-left:0px;		
}

QHeaderView::down-arrow { 
width: 4px; height:6px;
subcontrol-position: right;
top:-2px;
left:0px;
padding-right:10px;
} 

QHeaderView::up-arrow {
width: 4px; height:6px;
subcontrol-position: right;
top:-2px;
left:0px;
padding-right:10px;
} 




</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="autoExpandDelay">
      <number>-1</number>
     </property>
     <property name="indentation">
      <number>18</number>
     </property>
     <property name="animated">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>false</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
     <attribute name="headerMinimumSectionSize">
      <number>35</number>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>35</number>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MegaItemTreeView</class>
   <extends>QTreeView</extends>
   <header>MegaItemTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources_macx.qrc"/>
  <include location="Resources_macx.qrc"/>
 </resources>
 <connections/>
</ui>
